Extending explanation about port exclusions (#7927)
MicrosoftDocs/sql-docs: azure-sql/virtual-machines/windows/hadr-cluster-best-practices.md

- If the SQL Server database engine, Always On availability group listener, failover cluster instance health probe, database mirroring endpoint, cluster core IP resource, or any other SQL resource is configured to use a port between 49,152 and 65,536 (the [default dynamic port range for TCP/IP](/windows/client-management/troubleshoot-tcpip-port-exhaust#default-dynamic-port-range-for-tcpip)), add an exclusion for each port. Doing so will prevent other system process from being dynamically assigned the same port. The following example creates an exclusion for port 59999: 
`netsh int ipv4 add excludedportrange tcp startport=59999 numberofports=1 store=persistent`
It is necessary for the port to be excluded not being used when the exclusion is configured, otherwise, the execution will fail with a message like “The process cannot access the file because it is being used by another process.”
To confirm if the exclusions have been configured, we can check by using: `netsh int ipv4 show excludedportrange tcp`  

> By setting this exclusion for the AG role IP probe port, we could prevent an **Event ID: 1069** with status 10048, from Windows Failover cluster events, this is identified with event detail: 

Reviewer: ```suggestion
Setting this exclusion for the AG role IP probe port should prevent events such as **Event ID: 1069** with status 10048. This event can be seen in the Windows Failover cluster events with the following message:
```

```
Cluster resource '<IP name in AG role>' of type 'IP Address' in cluster role '<AG Name>' failed.
```

> An Event ID: 1069 with status 10048, can be identified from cluster logs with events like: 

Reviewer: ```suggestion
An Event ID: 1069 with status 10048 can be identified from cluster logs with events like: 
```
